iotedgedev/modules.py

```diff
@@ -301,7 +304,10 @@ def _get_launch_json(self, name, template, group_id):
             launch_json_content = Utility.get_file_contents(launch_json_file)
             for key, value in replacements.items():
                 launch_json_content = launch_json_content.replace(key, value)
-            launch_json = commentjson.loads(launch_json_content)
+            if PY2:
+                launch_json = commentjson.loads(launch_json_content.encode('ascii'))
```

Comment:
what is the default encoding? utf-8 cannot work?

Reply:
Since commentjson v0.7.2, we get "decoding Unicode is not supported" error on python 2.7. Other people also met this issue and I also shared our situation in the issue: https://github.com/vaidik/commentjson/issues/23
I haven't hear from the owner yet. Though limit the version of commentjson to < 0.7.2 can resolve this issue, I added this temporary solution to avoid possible dependency issue around the package.